app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -347,15 +356,24 @@ def parse_persistent_volume(persistent_volume)
       new_result = parse_base_item(persistent_volume)
       new_result.merge!(parse_volume_source(persistent_volume.spec))
       new_result.merge!(
-        :type           => 'PersistentVolume',
-        :parent_type    => 'ManageIQ::Providers::ContainerManager',
-        :capacity       => parse_resource_list(persistent_volume.spec.capacity.to_h),
-        :access_modes   => persistent_volume.spec.accessModes.join(','),
-        :reclaim_policy => persistent_volume.spec.persistentVolumeReclaimPolicy,
-        :status_phase   => persistent_volume.status.phase,
-        :status_message => persistent_volume.status.message,
-        :status_reason  => persistent_volume.status.reason
+        :type                    => 'PersistentVolume',
+        :parent_type             => 'ManageIQ::Providers::ContainerManager',
+        :capacity                => parse_resource_list(persistent_volume.spec.capacity.to_h),
+        :access_modes            => persistent_volume.spec.accessModes.join(','),
```

Comment:
@alongoldboim @zakiva I don't think this need a model change. Also please notice that on the UI side you do a ` @claim.desired_access_modes.join(',')` (why?).

I expect this to be an array (again, not sure if we need a db schema change).